lib/tools/adb-commands.js

```diff
@@ -530,40 +527,20 @@ methods.setDeviceProperty = async function (prop, val) {
   await this.shell(['setprop', prop, val]);
 };
 
-methods.getDeviceSysLanguage = async function () {
+methods.getDeviceLanguage = async function () {
   return await this.getDeviceProperty("persist.sys.language");
 };
 
-methods.setDeviceSysLanguage = async function (language) {
+methods.setDeviceLanguage = async function (language) {
   return await this.setDeviceProperty("persist.sys.language", language.toLowerCase());
 };
 
-methods.getDeviceSysCountry = async function () {
+methods.getDeviceCountry = async function () {
   return await this.getDeviceProperty("persist.sys.country");
 };
 
-methods.setDeviceSysCountry = async function (country) {
+methods.setDeviceCountry = async function (country) {
   return await this.setDeviceProperty("persist.sys.country", country.toUpperCase());
 };
 
-methods.getDeviceSysLocale = async function () {
```

Comment:
@imurchie I dont know how these changes get in.This PR doesn't need sysLocale,langugage changes.Will be fixing these now